suisa_sendemeldung/suisa_sendemeldung.py

```diff
@@ -194,13 +280,15 @@ def parse_filename(args, start_date):
     Returns:
         filename: the filename to use for the csv data
     """
+    print(args.station_name_short)
     if args.filename:
         filename = args.filename
     # depending on date args either append the month or the start_date
     elif args.last_month:
-        filename = __file__.replace(".py", "_{}.csv").format(start_date.strftime("%B"))
+        date_part = f"{start_date.strftime('%Y')}_{start_date.strftime('%m')}"
+        filename = f"{args.station_name_short}_{date_part}.csv"
     else:
-        filename = __file__.replace(".py", "_{}.csv").format(start_date)
+        filename = f"{args.station_name_short}_{start_date}.csv"
```

Comment:
`start_date` defaults to 2023-02-17 when printed. Is this expected?

Reply:
tbh, i didn't check closely since it was already using start_date before i added the station_name_short prefix feature and i just ported it 1-1.

the main reason for the file rename is for the last_month mode where there is an example in the Tarif's template that looks like `<station>_<year>_<month>.xslt`.

Reply:
`rabe-2023-02-17.xslx` doesn't sound too bad tho

Reply:
But in the test this piece of code returns a timestamp as well: https://github.com/radiorabe/suisa_sendemeldung/pull/189/files#diff-53beb436049dce8715db2c1afce4ee39c652198e83db9691009c9d6033e6aae8R107

Reply:
But yeah it's only a minor thing and is rarely used anyway.

Reply:
just realised that this does `rabe_2023-02-17 00:00:00.xslx`. let's fix this post-merge if we ever feel like it